app/utils/constants.ts

```diff
@@ -123,3 +124,288 @@ export const formatWikiURL = (page?: string) =>
   `https://mvp.developer.gov.bc.ca/docs/default/component/css-docs/${
     page ?? ''
   }?utm_source=sso-wiki&utm_medium=web&utm_campaign=retirement-notice-sso`;
+
+export const bcscPrivacyZones = () => {
```

Comment:
Do we still need these constants?

Reply:
@jlangy yes, given that BCSC endpoints are down, we still need to be able to show the options in that dropdown.